data/json/npcs/TALK_FACTION_CAMP.json

```
@@ -46,49 +68,91 @@
+    "responses": [
+      { "text": "What do you mean by angry?", "topic": "TALK_ALLY_TUTORIAL_MUTINY" },
+      { "text": "Tell me more about faction camps and food.", "topic": "TALK_CAMP_TUTORIAL_FOOD" },
+      { "text": "Tell me about building a camp.", "topic": "TALK_CAMP_TUTORIAL_BUILDING" },
+      { "text": "Give me some advice on building a camp.", "topic": "TALK_CAMP_TUTORIAL_ADVICE" },
+      { "text": "I know the basics.  Just tell me what changed.", "topic": "TALK_CAMP_TUTORIAL_CHANGES" },
+      { "text": "Never mind, let's go back to talking about camps.", "topic": "TALK_CAMP_GENERAL" },
+      { "text": "Never mind, let's talk about other things you can do.", "topic": "TALK_ALLY_TUTORIAL" },
+      { "text": "Never mind, let's talk about something else.", "topic": "TALK_FRIEND" },
+      { "text": "Forget it.  Let's go.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "TALK_CAMP_TUTORIAL_BUILDING",
+    "type": "talk_topic",
+    "dynamic_line": "Currently, there are three types of camps that you can start buildiing: modular field camps, fire station camps, and evac shelter camps.  There used to be a fourth called the primitive field camp, but you can't start those anymore, only upgrade them.\n  Modular field camps are the most flexible kind of camp since you can build them almost anywhere and can locate them to have plenty of space for expansions, but you start with nothing in a field and have to build every building, so they can require a lot of resources.  Fire station and evac shelter camps are faster to build since you start in an existing building, but you have to establish them in an existing building and there may not be any space for expansions.\n  Each camp location will have a variety of upgrade missions for it.  The various missions have descriptions.  In general, though, you'll need to establish housing if you want to expand, and some missions such as hunting or recruiting will require that you have some kind of kitchen or office that you can use to help schedule activities.",
```

buildiing -> building
```suggestion
    "dynamic_line": "Currently, there are three types of camps that you can start building: modular field camps, fire station camps, and evac shelter camps.  There used to be a fourth called the primitive field camp, but you can't start those anymore, only upgrade them.\n  Modular field camps are the most flexible kind of camp since you can build them almost anywhere and can locate them to have plenty of space for expansions, but you start with nothing in a field and have to build every building, so they can require a lot of resources.  Fire station and evac shelter camps are faster to build since you start in an existing building, but you have to establish them in an existing building and there may not be any space for expansions.\n  Each camp location will have a variety of upgrade missions for it.  The various missions have descriptions.  In general, though, you'll need to establish housing if you want to expand, and some missions such as hunting or recruiting will require that you have some kind of kitchen or office that you can use to help schedule activities.",
```